artemis/modules/sql_injection_detector.py

```diff
@@ -0,0 +1,272 @@
+import datetime
+import re
+import urllib
+from enum import Enum
+from timeit import default_timer as timer
+from typing import Any, Dict, List
+from urllib.parse import parse_qs, unquote, urlencode, urlparse, urlunparse
+
+import more_itertools
+from karton.core import Task
+
+from artemis import http_requests
+from artemis.binds import Service, TaskStatus, TaskType
+from artemis.config import Config
+from artemis.crawling import get_links_and_resources_on_same_domain
+from artemis.http_requests import HTTPResponse
+from artemis.karton_utils import check_connection_to_base_url_and_save_error
+from artemis.module_base import ArtemisBase
+from artemis.sql_injection_data import HEADER_KEYS, SQL_ERROR_MESSAGES, URL_PARAMS
+from artemis.task_utils import get_target_url
+
+
+class Statements(Enum):
+    sql_injection = 1
+    sql_time_base_injection = 2
```

Comment:
time_based, btw use string enum, it's easier to debug later (you don't have to lookup values here)